src/extension.ts

```diff
@@ -826,6 +874,7 @@ export class ExtensionManager implements vscode.Disposable {
             this.activeTestPresetSub = cmakeProject.onActiveTestPresetChanged(FireNow, p => {
                 this.statusBar.setTestPresetName(p?.displayName || p?.name || '');
             });
+            // no package/workflow info in status bar
```

Comment:
We need to make a single decision regarding whether we show this in the status bar. I think need to be able to. 

Currently in this code, we have mixed signals. I see in the code (and tested to confirm this is the case) that we allow for settings.json settings to configure the visibility of package and workflow settings in the status bar. However, this information here says that opposite.

Reply:
The spec says package/workflow should not be visible in status bar because limited space. After the latest FI I may have merged wrong some conflicts and I did not add correctly the settings part for package/workflow presets visibility.

Reply:
See https://github.com/microsoft/vscode-cmake-tools/pull/3548#discussion_r1467829227, I think to provide consistency, we will allow the user to put it in the status bar, but by default, it won't be there.